symphony/lib/toolkit/class.pagemanager.php

```diff
@@ -301,11 +301,11 @@ public static function editPageChildren($page_id = null, $page_path = null)
             );
 
             if (!PageManager::createPageFiles($page_path, $child['handle'], $child['path'], $child['handle'])) {
-                $success = false;
+                return false;
```

Comment:
It does change the behaviour. But I could not see how continuing to save things, while being technically in a error state, is a good thing. I thought about only issuing the calls and removing the if completely but that was not better.

Thinking about it, should we have 3 new boolean and return

```
return $successCreatePageFiles && $successEdit && $sucess
```

and perhaps rename $success ? That would not change the behaviour *per se*: only the return value... ?